src/entities/explorer/lib/utils.ts

```diff
@@ -0,0 +1,136 @@
+import { ParserField, Options } from 'graphql-js-tree';
+
+import { ITypeArguments, TypeGetTypeArgumentsReturn, IGetQueryValue } from '../model/';
+
+export const getTypeDetails = (parsedField: ParserField): TypeGetTypeArgumentsReturn => {
+  return parsedField.args.map((arg) => {
+    const fieldType = arg.type.fieldType;
+    if ('name' in fieldType) {
+      return {
+        name: arg.name,
+        type: `${fieldType.name}`,
+      };
+    } else if ('nest' in fieldType) {
+      if (fieldType.type === Options.required && 'nest' in fieldType.nest) {
+        if (fieldType.nest.type === Options.array && 'name' in fieldType.nest.nest) {
+          return {
+            name: arg.name,
+            type: `[${fieldType.nest.nest.name}]!`,
+          };
+        }
+      }
+      if (fieldType.type === Options.array && 'name' in fieldType.nest) {
+        return {
+          name: arg.name,
+          type: `[${fieldType.nest.name}]`,
+        };
+      }
+    }
```

Comment:
Way too much logic in the `map` method. Better split it into at least few functions and pass a single function to map